prop_Qt_x86.cmd

```diff
@@ -0,0 +1,22 @@
+@echo off 
+call "%ProgramW6432%\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd" /x86
+
+rem set PATH="%~dp0doxygen";%PATH%
+
+set _SDL_GEN="NMake Makefiles"
+set build_suffics="qt5_build"
+set _QT_PORT=1
+set SDL_ROOT="%CD%"
+set QTDIR=C:\Qt\Qt5.5.1\5.5\msvc2010
```

Comment:
So. As developer in required to install qt on disk C I think it is to the best solution. Also we will provide the code to Ford. And they will run it.
I would suggest to use some environment variable